### ISSUE TITLE
bug: a11y - Ionic select, Ionic date time picker read as button in scree readers

### ISSUE THREAD

**Issue by gurpreetonebcg** [ionic-team/ionic-framework#23600]
# Bug Report

**Ionic version:**

```
@ionic/angular 5.6.0
```

**Current behavior:**

When your focus on select & datetime picker it will read as button.

**Expected behavior:**

It should read as input select.

**Steps to reproduce:**

Take the code from attached link or install .apk provide in attached link, start talk back screen reader open app and navigate to input select on the page you will observe that screen reader is announcing it as button.

[Download .apk here](https://drive.google.com/file/d/1IOanXH8NcaVeYhiTrfjEveQBrpMH8IhI/view?usp=sharing)

**Related code:**

[Downlead code here](https://drive.google.com/file/d/1TfR-9kxiYYeVQkr6WOEjHaSuN_ZIq9my/view?usp=sharing) 

**Other information:**

[Screen short of issue](https://drive.google.com/file/d/1MUNFPxVZZB69E8Iz9erdV7Nv0Ekj_wBS/view?usp=sharing)

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.11.8
   Ionic Framework               : @ionic/angular 5.6.0
   @angular-devkit/build-angular : 0.1101.4
   @angular-devkit/schematics    : 11.1.4
   @angular/cli                  : 11.1.4
   @ionic/angular-toolkit        : 3.1.0

Cordova:

   Cordova CLI       : 10.0.0
   Cordova Platforms : android 9.0.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 4 other plugins)

Utility:

   cordova-res : 0.11.0
   native-run  : 1.0.0
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you please clarify why these elements should be read as `input select`? The elements that are being focused in your application are not the actual select and date pickers, they are buttons that can be clicked to present the select and date pickers.

**Comment by gurpreetonebcg** [comment]
Thanks for reply, 

As per the WCAG guideline 1.3.1 Info and Relationships, 4.1.2 Name, Role, Value the info & the relation should be clear between the two elements & there roles.

Here when screen reader announces input as button, it confuses the user that there will be some action performed by the clicking on that button, but in actual user has to select the data to perform action so the roles should be clear & these inputs should behave as select.

**Comment by liamdebeasi** [comment]
Thanks for the additional information. Are you referring to the buttons that open the select/datetime dialogs? Or are you referring to the select/datetime dialogs themselves?

**Comment by gurpreetonebcg** [comment]
Yes am referring to the buttons that opens date Time dialogs. please refer to attached video the video https://drive.google.com/file/d/1M8cNA7a8Ci-mdoO24oWDTObTthrINZ9e/view?usp=sharing
